<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PacketCapture</class>
 <widget class="QWidget" name="PacketCapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>551</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="MyLabel" name="labelIp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="MyLineEdit" name="lineEditIP">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="MyLabel" name="labelPort">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="MyLineEdit" name="lineEditPort"/>
     </item>
     <item row="2" column="0">
      <widget class="MyLabel" name="labelNic">
       <property name="text">
        <string>NIC</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ComboBox" name="comboBoxNic"/>
     </item>
     <item row="3" column="0">
      <widget class="MyLabel" name="labelExportDirectory">
       <property name="text">
        <string>Export Directory</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QWidget" name="" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="MyLineEdit" name="lineEditExportDirectory">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyPushButton" name="pushButtonBrowse">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>369</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButtonEnd">
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButtonBack">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
